docs/3-copilot-agent-mode-vscode.md

```diff
@@ -26,7 +26,7 @@ The website is a crowdfunding platform for board games with a developer theme. I
 
 To make running the website easier, we have provided a script that will start both the front-end and back-end servers. You can run this script in your GitHub Codespace to start the website with the following instructions:
 
-1. Return to your codespace.
+1. Return to your codespace. Continue working in your current branch.
```

Comment:
```suggestion
1. Return to your codespace. We'll continue working in your current branch.
```

I see what's being done, but this still feels a little clunky. Going to leave it with the small update, but it's just a bit off center for me.